src/lib/gundb/UserStorage.js

```diff
@@ -41,6 +41,21 @@ export type TransactionEvent = FeedEvent & {
   }
 }
 
+const getReceiveDataFromReceipt = receipt => {
+  const transferLog = receipt.logs.find(log => {
+    const { events } = log
+    const eventIndex = events.findIndex(
+      event => event.name === 'to' && event.value === '0xcb07bf5869a9f4f17ad7cc07c5c15deb5ac665fb'
```

Comment:
This hard coded value seems problematic...

Reply:
:man_facepalming:  my bad! -> fixed @sirpy